pkg/routeversion/version.go

```diff
@@ -0,0 +1,74 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package routeversion
+
+import (
+	"fmt"
+	"runtime"
+
+	"k8s.io/component-base/metrics"
+	"k8s.io/component-base/metrics/legacyregistry"
+
+	"k8s.io/apimachinery/pkg/version"
+)
+
+var (
```

Comment:
None of these variables get populated given the [build-machine-go assumes](https://github.com/openshift/openshift-controller-manager/blob/master/vendor/github.com/openshift/build-machinery-go/make/lib/golang.mk#L69) only `$(GO_PACKAGE)/pkg/version` path. Nevertheless, once the code is moved into the new repository, the version path gets normalized and properly populated again.

Reply:
ack, I think we can live with the broken version for the time of the move